frontend_app_logbook/src/component/Logbook.jsx

```diff
@@ -157,7 +157,7 @@ export class Logbook extends React.Component {
 
   async save (newLogbook) {
     const { props } = this
-    newLogbook.entries.toSorted((a, b) => new Date(b.deadline) - new Date(a.deadline))
+    newLogbook.entries.toSorted((a, b) => new Date(b.datetime) - new Date(a.datetime))
```

Comment:
I'm confused, I don't think this does anything since array.toSorted is immutable

Reply:
Indeed, I might have changed it without thinking :/ will fix it when the tests will end, so I don't have to wait for them again

Reply:
fixed